sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -67,4 +67,7 @@ src_compile() {
 
 src_install() {
 	newbin ${GOBIN}/internal ${PN}
+
+	exeinto "/usr/libexec"
+	newexe ${GOBIN}/internal "${PN}-rmcfg"
```

Comment:
Does this file end up in the initramfs? We need to be careful because the arm64 initramfs almost hits the size limit.

Reply:
Yeah it does not. The initramfs is being built in the `sys-kernel/coreos-kernel` emerging, that's why we have a dependency for Ignition in the ebuild: https://github.com/flatcar-linux/coreos-overlay/blob/927d1afcc9feb157fd8c005a3d22b9fb3c9121a9/sys-kernel/coreos-kernel/coreos-kernel-5.15.44.ebuild#L20

The Ignition dracut's module setup explicitly installs the `ignition` binary: https://github.com/flatcar-linux/bootengine/blob/39caefd1da35d3600120b44ea5ee23ed22129b46/dracut/30ignition/module-setup.sh#L77-L78 so we're safe - `ignition-rmcfg` will never land in the initramfs as long as we don't explicitly pulling it.